### PR TITLE
Refer to the latest published version of the WebNN API

### PR DESCRIPTION
**Pull request by ibelem**
As the follow up of https://github.com/webmachinelearning/webmachinelearning.github.io/pull/36, updated all the spec link to latest published version.

@anssiko @Honry PTAL

### CODE REVIEW

File: _layouts/community.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+<html lang="{{ page.lang | default: site.lang | default: " en" }}">
```
Comment (Honry) [inline _layouts/community.html]: Nit: unnecessary whitespace.

File: _layouts/faq.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+<html lang="{{ page.lang | default: site.lang | default: " en" }}">
```
Comment (Honry) [inline _layouts/faq.html]: Ditto.

File: _posts/2020-03-17-noise-suppression-net-v2.md [hunk 3]
```diff
@@ -30,39 +30,104 @@ export class NSNet2 {
     const nn = navigator.ml.getNeuralNetworkContext();
```
Comment (Honry) [inline _posts/2020-03-17-noise-suppression-net-v2.md]: This example code is outdate, could you please update it to the latest one at https://github.com/webmachinelearning/webnn-samples/blob/master/nsnet2/nsnet2.js#L9?

File: _layouts/community.html (hunk 1 above)
Comment (ibelem) [reply _layouts/community.html]: Thanks @Honry, fixed.

File: _layouts/faq.html (hunk 2 above)
Comment (ibelem) [reply _layouts/faq.html]: Thanks @Honry, fixed.

File: _posts/2020-03-17-noise-suppression-net-v2.md (hunk 3 above)
Comment (ibelem) [reply _posts/2020-03-17-noise-suppression-net-v2.md]: Thanks @Honry, fixed.

**Review by Honry — APPROVED** [review]
LGTM, thanks!

**Review by anssiko — APPROVED** [review]
Thanks @honry and @ibelem!
